src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -154,171 +155,179 @@ private RootCommand SetupCommandLineAPI()
             return rootCommand;
         }
 
-        private int AnalyzeTemplate(FileInfo templateFilePath, FileInfo parametersFilePath, FileInfo configurationsFilePath, ReportFormat reportFormat, FileInfo outputFilePath, bool runTtk, bool verbose, bool printMessageIfNotTemplate = true, IReportWriter writer = null, bool readConfigurationFile = true, ILogger logger = null)
+        private int AnalyzeTemplateCommandHandler(
```

Comment:
SummarizeLogs now is only called in the directory handler, I think logs should still be summarized for individual templates also

Reply:
The way I understood the summary logger was to make sure all the errors and warnings that occurred while analyzing several templates in a directory would be clearly presented after all analysis was over.  In the case of a single template, those errors/warnings would just be immediately repeated.  I didn't see any value in doing this, and felt it would be a little annoying instead.

Reply:
I guess it depends on the template size/number of results. I can see a single template generating a console output big enough for the summary to make sense anyways

Reply:
Keeping summary with template analysis.